### PR TITLE
Update "Getting Started" tutorial and README

### PR DESCRIPTION
**Pull request by jwitz**
# Description
## What is the current behavior?

Resolves https://github.com/astronomer/astro-sdk/issues/652

## What is the new behavior?

This PR updates the style and wording of our "Getting Started" documentation without disrupting the core content. I tried to make it easier to follow the quickstarts and to understand the benefits of the SDK

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [X] Extended the README / documentation, if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,114 +11,119 @@
 [![PyPI downloads](https://img.shields.io/pypi/dm/astro-sdk-python.svg)](https://pypistats.org/packages/astro-sdk-python)
 [![Contributors](https://img.shields.io/github/contributors/astronomer/astro-sdk)](https://github.com/astronomer/astro-sdk)
 [![Commit activity](https://img.shields.io/github/commit-activity/m/astronomer/astro-sdk)](https://github.com/astronomer/astro-sdk)
-[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/astronomer/astro-sdk/main.svg)](https://results.pre-commit.ci/latest/github/astronomer/astro-sdk/main)
```
Comment (tatiana) [inline README.md:14]: @jwitz any reason for removing this?

File: README.md [hunk 2]
```diff
@@ -144,41 +149,44 @@ How to Train Your Dragon 2 (2014)|7.9
...
-  * ignore: do not add rows that already exist
-  * update: replace existing rows with new ones
-* `export_file`: export SQL table rows into a destination file
-* `dataframe`: export given SQL table into in-memory Pandas data-frame
+
+The following are some key functions available in the SDK:
+
+- `load_file`: load a given file into a SQL table
+- `transform`: applies a SQL select statement to a source table and saves the result to a destination table
+- `truncate`: remove all records from a SQL table
```
Comment (tatiana) [inline README.md:157]: We probably forgot to update this in the past, `truncate` is now `drop_table` and it actually drops the table - could you fix this as part of this PR?

**Review by tatiana — COMMENTED** [review]
@jwitz it's looking great overall, thank you very much! Just two minor comments!

File: README.md [hunk 3]
```diff
@@ -11,114 +11,119 @@
...
-airflow db init
-```
+The Astro Python SDK is available at [PyPI](https://pypi.org/project/astro-sdk-python/). Use the standard Python
+[installation tools](https://packaging.python.org/en/latest/tutorials/installing-packages/).
 
-Create an SQLite database for the example to run with and run the DAG:
+To install a cloud-agnostic version of the SDK, run:
 
 ```shell
-# The sqlite_default connection has different host for MAC vs. Linux
```
Comment (utkarsharma2) [inline README.md:95]: is this issue solved in airflow?

File: README.md (hunk 2 above)
Comment (utkarsharma2) [reply README.md:157]: Can we link this section to readthedocs link - https://astro-sdk.readthedocs.io/en/latest/ operator section?

**Review by kaxil — APPROVED** [review]

**Pull request merged**